packages/tools/src/tools/segmentation/SphereScissorsTool.ts

```diff
@@ -118,7 +118,9 @@ class SphereScissorsTool extends BaseTool {
     const { representationData } = getSegmentation(segmentationId);
 
     // Todo: are we going to support contour editing with rectangle scissors?
-    const { volumeId } = representationData[type] as LabelmapSegmentationData;
+    const { volumeId } = representationData[
+      type
+    ] as LabelmapSegmentationDataVolume;
```

Comment:
I guess this requires a Todo that says `handle stack later`?

Reply:
yes